libs/ui/src/components/Explorer/Explorer.tsx

```diff
@@ -26,9 +26,13 @@ import {
 import {useSearchInput} from './useSearchInput';
 import {usePagination} from './usePagination';
 import {Loading} from '../Loading';
+import {Empty} from 'antd';
 
 const isNotEmpty = <T extends unknown[]>(union: T): union is Exclude<T, []> => union.length > 0;
 
+const emptyArray = [];
```

Comment:
**note:** [] or {} should not be used directly in components properties as they lead to new reference at each render. Also true if declared as const inside the component. The solution is to declare as constants outside, so the reference is always the same